pyproject.toml

```diff
@@ -6,7 +6,7 @@ build-backend = "setuptools.build_meta"
 name = "posit-sdk"
 description = "Posit SDK for Python"
 readme = "README.md"
-requires-python = ">=3.8"
+requires-python = ">=3.9"
```

Comment:
Customers often have content running on older versions, and it's not (yet) trivial within Connect for them to "just" upgrade Python. So, if we stop supporting older versions in Connect, the effect is that it prevents customers from upgrading Connect because their content would stop working. We're working on a few things to make this better, but until we offer a better path for these customers, we're not dropping Python versions anymore.

The SDK doesn't necessarily need to follow what Connect does, but there are arguments for keeping it more permissive so that it is more widely usable by customers who haven't upgraded yet.

Reply:
Thank you!

I'll add it back as it was more build tool driven, rather than package code requirements